modules/conda-setup.sh

```
@@ -17,12 +17,11 @@ bash Mambaforge-$MAMBAFORGE_VERSION-Linux-x86_64.sh -b -f -p ~/conda
 rm -f Mambaforge-$MAMBAFORGE_VERSION-Linux-x86_64.sh
 popd
 
-BASE_ENVIRONMENT_YML="$SETUPDIR/conda-environment.yml"
-if [ -f "$BASE_ENVIRONMENT_YML" ]; then
-    echo "Conda: Configuring shell and installing packages"
-    # Activates the conda environment in bashrc
+echo "Searching for conda environments"
+
+compgen -G "${SETUPDIR}/conda-environment*.yml" && \
     ~/conda/bin/mamba init
```

```
$ compgen -G tmp/nonexistent.yml && echo 1
$ echo $?
1
```

so this will abort the Packer build if there's no conda env instead of skipping